camelot/handlers.py

```diff
@@ -3,7 +3,7 @@
 import os
 import sys
 
-from PyPDF2 import PdfFileReader, PdfFileWriter
+from PyPDF2 import PdfReader, PdfWriter
```

Comment:
pypdf 3.2.0 use all-lowercase name, maybe upgrade to pypdf 3.2.0, and not using PyPDF2 anymore?

Reply:
I think that would be a good idea.